chopper_built_value/test/converter_test.dart

```diff
@@ -33,29 +33,31 @@ void main() {
 
     test('convert response with wireName', () {
       final string = '{"\$":"DataModel","id":42,"name":"foo"}';
-      Response response = Response(http.Response(string, 200), string);
-      response = converter.convertResponse<DataModel, DataModel>(response);
+      final response = Response(http.Response(string, 200), string);
+      final convertedRes =
```

Comment:
We should not save on character count like this. 😉 

`convertedRes` -> `convertedResponse`

Reply:
Of course, this applies to all occurrences in this PR.

Reply:
I have fixed this :)